### PR TITLE
Account pages

### PR DESCRIPTION
**Pull request by HSEshan**
Added the signup and login pages for user accounts. We can work on the design a bit more.
Update: Added our own CSS sheet, improved file name readability
Update2: Modified profilePage layout to be consistent with other pages

### CODE REVIEW

File: codeHelp.css
Comment (23460542) [inline codeHelp.css:1]: Maybe change name of css doc to custom.css or extra.css to make it more obvious that this is extra CSS code auxiliary to bootstrap CSS.

File: navBar for all pages.html
Comment (23460542) [inline navBar for all pages.html:1]: This separate navbar HTML will be the base.html in flask branch. Buttons should be changed to <a> though as they are links and the styling will still look the same.

File: codeHelp.css
Comment (HSEshan) [reply codeHelp.css:1]: Good idea

File: navBar for all pages.html
Comment (HSEshan) [reply navBar for all pages.html:1]: The changes will be implemented as we progress

**Pull request merged**